internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -596,18 +576,6 @@ func (r *AddOnDefinitionResource) Update(ctx context.Context, req resource.Updat
 	}
 	plan.Labels = labels
 
-	if addOnDefinition.TerraformCloudWorkspaceDetails != nil {
```

Comment:
This was accidentally included in a merged PR so I am removing it so it's back to how it used to function.

Reply:
Ditto here as well about not wanting to delete the whole block.